packages/orchestration/tools/ibc-mocks.ts

```diff
@@ -165,6 +165,7 @@ type BuildVTransferEventParams = {
   sourceChannel?: IBCChannelID;
   /* support bigint and string, to facilitate bootstrap testing */
   sequence?: PacketSDKType['sequence'] | JsonSafe<PacketSDKType['sequence']>;
+  memo?: string;
```

Comment:
not worth changing but I'd have made this a `feat` since `tools` are exported from the package